types/levelup/index.d.ts

```diff
@@ -167,8 +169,5 @@ export interface LevelUpChain<K = any, V = any> {
     write(): Promise<this>;
 }
 
-export const errors: LevelUpConstructor["errors"];
-
-export const LevelUp: LevelUpConstructor;
-
-export {};
+declare const levelup: LevelUpConstructor;
+export default levelup;
```

Comment:
Ref: https://github.com/Level/levelup/blob/e73f536a46d7b8b2fa3d9b5abd0ae6efca41653e/lib/levelup.js#L333

Line 333:
```js
module.exports = LevelUP
```

In order to match the export statement, the `export =` syntax should be used. 
https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require

example: 
```ts
export default levelup
```